test/e2e/framework/networking_utils.go

```diff
@@ -218,8 +218,9 @@ func (config *NetworkingTestConfig) DialFromContainer(protocol, containerIP, tar
 // pod and confirm it doesn't show up as an endpoint.
 func (config *NetworkingTestConfig) DialFromNode(protocol, targetIP string, targetPort, maxTries, minTries int, expectedEps sets.String) {
 	var cmd string
+	// Make the command timeout after 1s (no matter what protocol it is).
 	if protocol == "udp" {
-		cmd = fmt.Sprintf("echo 'hostName' | timeout -t 3 nc -w 1 -u %s %d", targetIP, targetPort)
+		cmd = fmt.Sprintf("echo 'hostName' | timeout -t 1 nc -w 1 -u %s %d", targetIP, targetPort)
```

Comment:
seems weird, how much does this 1s actually buy us? we're only retrying on some bounded set of endpoints right? I'm fine with it if we really need it. Please add a comment saying when we can expect the terminated message in stderr.